perfect-cell.cydsn/atlas_wq_sensor.h

```diff
@@ -30,13 +32,13 @@ struct{
 int atlas_sensor_sleep(uint8 sensor_address);
 
 /**
- * @brief Wake Atlas I2C sensor from sleep mode
+ * @brief Calibrate Atlas sensor using single-point method
  *
  * @param sensor_address I2C slave address of sensor
  *
  * @return 1 on success
  */
-int atlas_sensor_wake(uint8 sensor_address);
+int atlas_sensor_calibrate(uint8 sensor_address);

```

Comment:
I don't think we need to calibrate every time we wake sensor up. This would probably change the results. We should only be calibrating with the solutions. Add sleep command would not effect the calibration.